src/components/views/Booktrak/BooktrakUtils.tsx

```diff
@@ -0,0 +1,49 @@
+import { ModelsBookListing } from "wso-api-client/lib/services/types";
+
+export const BookConditionEnumToString = (
+  condition: ModelsBookListing.ConditionEnum
+): string => {
+  const ListingConditionEnum = ModelsBookListing.ConditionEnum;
+  switch (condition) {
+    case ListingConditionEnum.Empty:
```

Comment:
nit: Move this case to the bottom and remove its return since it's identical to default case.